stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/AttachmentSlidePagerAdapter.kt

```diff
@@ -0,0 +1,19 @@
+package io.getstream.chat.android.ui.images
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentActivity
+import androidx.viewpager2.adapter.FragmentStateAdapter
+
+internal class AttachmentSlidePagerAdapter(
+    fa: FragmentActivity,
+    private val imageList: List<String>
+) : FragmentStateAdapter(fa) {
+
+    override fun getItemCount(): Int = imageList.size
+
+    override fun createFragment(position: Int): Fragment {
+        return ImageSlidePageFragment().apply {
```

Comment:
Do we support device rotation? This won't work on configuration change

Reply:
Even though the adapter is recreated on rotation, the fragments are restored from bundle